### PR TITLE
Release 0.6.1

### PR DESCRIPTION
**Pull request by jrezai**

### CODE REVIEW

File: org.lvnauth.LVNAuth.metainfo.xml
Comment (hfiguiere) [inline org.lvnauth.LVNAuth.metainfo.xml:1]: why is this file added? it's never used.

Comment (jrezai) [reply org.lvnauth.LVNAuth.metainfo.xml:1]: I thought it was needed for the release notes. How do the release notes get added without this file?
It's my first Flathub update, I've never updated my app here before.
Thanks.

Comment (hfiguiere) [reply org.lvnauth.LVNAuth.metainfo.xml:1]: > I thought it was needed for the release notes. How do the release notes get added without this file? 

In the file that is actually installed.... aka not this one.

Comment (jrezai) [reply org.lvnauth.LVNAuth.metainfo.xml:1]: Oh I think I understand now.
Thanks.
